doc/motion_planning_api/launch/motion_planning_api.launch.py

```diff
@@ -0,0 +1,198 @@
+import os
```

Comment:
FYI these launch files are likely to get a lot simpler when this PR lands:  https://github.com/ros-planning/moveit2/pull/591

I think they're fine for now, though

Reply:
Also, I think you should rename this file `setup.launch.py`. That would be a bit more descriptive.

It's hard for me to tell the difference between `motion_planning_api.launch.py` and `motion_planning_api_tutorial.launch.py`

(don't forget to update the filename in the doc, too)

Reply:
I renamed it to `motion_planning_api_setup.launch.py`.  I think that calling it just `setup.launch.py` could conflict with other tutorials, since all the launch files from all of the tutorials are installed to the same directory.